P9Operators/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿
+// Execise 10
+/*
+int TwentyFourHourSeconds = 86400;
+
+Console.WriteLine("Give me a number of sec.");
+float userInput = float.Parse(Console.ReadLine());
+
+// in total days form
+float inTotal = userInput / TwentyFourHourSeconds;
+
+// days, hour, min & sec
+int intInput = (int)userInput;
+
+int day = intInput / TwentyFourHourSeconds;
+int oneDayHours = day * 24;
+int hours = intInput / 60 / 60 - 24;
```

Comment:
No, this maths here doesn't check out for input where for example the user enters a time of more than 48 hours. Test it for for example 345678, it will show way more than 23 hours.

This is, why it's important to test our code for several different inputs.